src/remote.js

```diff
@@ -194,64 +202,75 @@
             );
           },
           function(error) {
-            util.err('Unable to create monaca project: ' + error);
+            util.err('Unable to create monaca project: ' + JSON.stringify(error));
```

Comment:
I think we should display error message instead of serializing a JSON object to a string.